Updated the ansible-galaxy docs

anis016 opened the pull request:
<!--- Your description here -->

Updated the documentation that reflects the default branch that will be used for the `version` option in ansible-galaxy.

+label: docsite_pr

##### SUMMARY
Updated the documentation that reflects the default branch that will be used for the `version` option in ansible-galaxy. Fixes #51748 

##### ISSUE TYPE
<!--- Pick one below and delete the rest -->
- Docs Pull Request

##### COMPONENT NAME
docs.ansible.com

##### ADDITIONAL INFORMATION
<!--- Include additional information to help people understand the change here -->
<!--- A step-by-step reproduction of the problem is helpful if there is no related issue -->

<!--- Paste verbatim command output below, e.g. before and after your change -->
```paste below

```

anis016 committed:
Updated the ansible-galaxy docs

<!--- Your description here -->

Updated the documentation that reflects the default branch that will be used for the `version` option in ansible-galaxy.

+label: docsite_pr

Akasurde commented:
@anis016 Thanks for the PR. I can see this block code https://github.com/ansible/ansible/blob/6d4307377fca671e2abba082f00167c2b63beedf/lib/ansible/galaxy/role.py#L237 where it set it master. So might want to rephrase the documentation.

anis016 committed:
Updated the documentation

As per the code (https://github.com/ansible/ansible/blob/6d4307377fca671e2abba082f00167c2b63beedf/lib/ansible/galaxy/role.py#L237)[ansible/galaxy/role.py] Defaults to the branch set as a default in the repository otherwise defaults to the master.

anis016 commented:
@Akasurde thank you! I checked the code and updated the documentation accordingly.

samccann commented on docs/docsite/rst/reference_appendices/galaxy.rst:
```diff
@@ -99,7 +99,7 @@ Each role in the file will have one or more of the following attributes:
      Specify the SCM. As of this writing only *git* or *hg* are supported. See the examples below. Defaults to *git*.
    version:
-     The version of the role to download. Provide a release tag value, commit hash, or branch name. Defaults to *master*.
+     The version of the role to download. Provide a release tag value, commit hash, or branch name. Defaults to the branch set as a default in the repository otherwise defaults to the *master*.
```
```suggestion
     The version of the role to download. Provide a release tag value, commit hash, or branch name. Defaults to the branch set as a default in the repository, otherwise defaults to the *master*.
```

samccann reviewed (commented):
Thanks for the addition!  I added a minor comma nit that you should be able to 'accept suggestion' to include it in this patch.

samccann committed:
Update docs/docsite/rst/reference_appendices/galaxy.rst

Co-Authored-By: anis016 <anis016@users.noreply.github.com>

samccann commented:
Thanks @anis016 for the ansible documentation fix!
